README.md

```diff
@@ -1 +1,101 @@
-# Dash Template
+### Interested in contributing a code example?
+
+Please take a look at our [contribution guidelines](CONTRIBUTING.md) before
+getting started. Thank you!
+
+The Dash template is intended for demos that would benefit from a user interface. This user interface could include settings to run and customize the problem, an interactive graphical element, or tables/charts to compare different solutions. This template is also useful for demos that are intended for a general audience, as it is more approachable for those without a technical background.
+
+<!-- Before submitting your code, please delete everything above and including this line. -->
+
+# Demo Name
+
+Describe your demo and specify what it is demonstrating. Consider the
+following questions:
+
+* Is it a canonical problem or a real-world application?
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target: beginner, advanced?
+
+A clear description allows us to properly categorize your demo.
+
+Please include a screenshot of your demo below.
+
+![Demo Example](static/demo.png)
+
+## Installation
+You can run this example without installation in cloud-based IDEs that support the [Development Containers specification](https://containers.dev/supporting) (aka "devcontainers").
+
+For development environments that do not support `devcontainers`, install requirements:
+
+```bash
+pip install -r requirements.txt
+```
+
+If you are cloning the repo to your local system, working in a [virtual environment](https://docs.python.org/3/library/venv.html) is recommended.
+
+## Usage
+Your development environment should be configured to access the [Leap&trade; Quantum Cloud Service](https://docs.ocean.dwavesys.com/en/stable/overview/sapi.html). You can see information about supported IDEs and authorizing access to your Leap account [here](https://docs.dwavesys.com/docs/latest/doc_leap_dev_env.html).
+
+Run the following terminal command to start the Dash app:
+
+```bash
+python app.py
+```
+
+If you plan on editing any files while the app is running, please run the app with the `--debug` command-line argument for live reloads and easier debugging:
```

Comment:
This is important to the contributor but might confuse beginner-level end users. 
Consider removing this debug section from the README: someone modifying the code can be alerted to this option through comment lines in the code file instead